vars/provisionNodes.groovy

```diff
@@ -83,7 +85,7 @@ def call(Map config = [:]) {
      }
   }
   if (env.REPOSITORY_URL != null) {
-    if (distro.startsWith("el7")) {
+    if (distro.startsWith("centos7")) {
```

Comment:
Shouldn't these be family matches and not distro matches?  There is really no reason at all for us to (potentially) be building RHEL and SLES specific binaries as we are assuming bit-for-bit compatibility between the commercial distros and their FOSS counterparts.

So rather than changing these from _family_ + _version_, probably we want to change the _leap15_ tests to _family_ + _version_.  We probably also don't really have a lot of use for *SLES_15* specific repos distinct from *LEAP_15*.  At least we know of no reason to have such at this time any more than we have *RHEL_7* repos vs. *CENTOS_7* (which we are already referring to as *EL_7*).